Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -681,7 +700,7 @@ def check_for_unanswered_questions():
         question['last_poll_time'] = now_string
         updated_questions.append(question)
     if updated_questions:
-        set_to_integration_context_with_retries({'users': users, 'questions': questions}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+        set_to_integration_context_with_retries({'questions': questions}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
```

Comment:
We weren't even using the Users value in this flow. No reason to pull it from context and store in memory.